client/packages/common/src/ui/components/inputs/SearchBar/SearchBar.tsx

```diff
@@ -38,14 +46,18 @@ const EndAdornment: FC<{
   );
 };
 
-export const SearchBar: FC<SearchBarProps> = ({
+export const SearchBar = ({
   value,
   onChange,
+  onSearchIconClick,
```

Comment:
I would only add `onClear` method and `alwaysShowClear`, keep everything else as is